src/AppBundle/Utils/Markdown.php

```diff
@@ -25,7 +25,10 @@ class Markdown
 
     public function __construct()
     {
-        $this->parser = new \Parsedown();
+        $parsedown = new \Parsedown();
```

Comment:
For simplicity, the `AppBundle\Utils\Markdown` is already service wrapper for `Parsedown` and to have `new` in its constructor is acceptable IMO.